ncollide_geometry/query/point_internal/point_mesh.rs

```diff
@@ -108,10 +148,12 @@ impl<P: Point, M: Isometry<P>> PointQuery<P, M> for TriMesh<P> {
     }
 }
 
+
 impl<P: Point, M: Isometry<P>> PointQuery<P, M> for Polyline<P> {
     #[inline]
     fn project_point(&self, m: &M, point: &P, solid: bool) -> PointProjection<P> {
-        self.base_mesh().project_point(m, point, solid)
+        let (projection, _) = self.project_point_with_extra_info(m, point, solid);
+        projection
```

Comment:
You could just return `self.project_point_with_extra_info(m, point, solid).0` here.